docker/pypi/wmagent/Dockerfile

```diff
@@ -3,26 +3,33 @@ MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
 
 # TAG to be passed at build time through `--build-arg TAG=<WMA_TAG>`. Default: None
 ARG TAG=None
-ENV WMA_TAG=${TAG}
+ARG WMA_TAG=$TAG
```

Comment:
Since there are lots of WMA_XXX env variables I suggest to put them into separate section in README file and provide proper documentation about them over there.

Reply:
good point. I will do that

Reply:
done